cli/Cargo.toml

```diff
@@ -53,6 +53,7 @@ semver = "1.0.4"
 once_cell = "1.9.0"
 
 [dev-dependencies]
+foundry-utils = { path = "./../utils", features = ["test"] }
```

Comment:
Looks unused everywhere?

Reply:
right, but it's helpful when debugging ethers-solc related things, simply add the init_trace function to a test and run it.